### ISSUE TITLE
Remove all references to Mandel

### ISSUE THREAD

**Issue by arhag** [AntelopeIO/DUNE#1]
Remove all references to `Mandel` or `mandel` from within the `antelope-main` branch of `DUNE`.

In addition, only one reference to `eosnetworkfoundation` should exist in the code: in the line within `bootstrap_env.sh` that clones the system contracts repo that is located in the `eosnetworkfoundation` organization. This is because DUNE at this point expects the EOS system contracts managed by the ENF (not a more limited reference system contracts which are appropriate to include in `AntelopeIO`). We may want to revisit this in the future for DUNE.

~This means that the `ORG` variable in `bootstrap_env.sh` should be updated to `AntelopeIO` and the `experimental-binaries` containing Leap and CDT packages should come from the `AntelopeIO` organization. It is probably best to wait until Leap 3.1.0-rc4 and CDT 3.0.0-rc2 releases are made and their binary packages made available before starting on this issue to have the necessary prerequisites to ensure that DUNE works correctly after making the changes required by this issue.~ (This part was left for another issue: https://github.com/AntelopeIO/DUNE/issues/3.)

The `mandel-contracts` references will need to be replaced by `system-contracts`. That means this issue is blocked on the dependency of the `eosnetworkfoundation/mandel-contracts` repo first being renamed to `system-contracts`.
